### ISSUE TITLE
Thready Safety Issue (Round 2)

### ISSUE THREAD

**Issue by Kashomon** [pires/hazelcast-kubernetes-bootstrapper#10]
Looks like the thread safety issue didn't get fixed in the latest refactor. The offending line of code is still there:

```java
        final TcpIpConfig tcpCfg = new TcpIpConfig();
        nodes.parallelStream().forEach(tcpCfg::addMember);
```

**Comment by pires** [comment]
Sorry @Kashomon I had a bad merge and feel so dumb right now.

**Comment by pires** [comment]
Fixed and released version 0.6.1.

**Comment by Kashomon** [comment]
No worries! Thanks for the fix.
